projects.json

```diff
@@ -4329,5 +4329,31 @@
          "tags": "swiftpm"
        }
      ]
+   },
+   {
+     "repository": "Git",
+     "url": "https://github.com/LiarPrincess/Violet",
+     "path": "violet",
+     "branch": "main",
+     "maintainer": "mail@liarprincess.me",
+     "compatibility": [
+       {
+         "version": "5.3",
```

Comment:
Can we update the version to Swift 5.0? We are trying to keep the version 4.0, 4.2, and 5.0